src/modules/function/declaration.rs

```diff
@@ -317,13 +318,20 @@ impl FunctionDeclaration {
             result.push(String::from("```\n"));
             if test_path.exists() && test_path.is_dir() {
                 if let Ok(entries) = fs::read_dir(test_path) {
-                    let pattern = format!("{}*.ab", self.name);
-                    let pattern = glob::Pattern::new(&pattern).unwrap();
+                    let pattern1 = format!("{}*.ab", self.name);
+                    let pattern2 = format!("{}_{}*.ab", lib_name, self.name);
+                    let pattern1 = glob::Pattern::new(&pattern1).unwrap();
+                    let pattern2 = glob::Pattern::new(&pattern2).unwrap();
```

Comment:
Consider using blocks
```suggestion
                    let pattern1 = {
                        let pat = format!("{}*.ab", self.name);
                        glob::Pattern::new(&pattern1).unwrap()
                    };  
                    let pattern2 = {
                        let pat = format!("{}_{}*.ab", lib_name, self.name);
                        glob::Pattern::new(&pattern2).unwrap()
                    };
```